FetchTests/ResponseErrorTests.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ResponseErrorTests.swift
+//  Fetch-iOS
+//
+//  Created by David Hardiman on 02/01/2018.
+//  Copyright © 2018 David Hardiman. All rights reserved.
+//
+
+@testable import Fetch
+import Nimble
+import XCTest
+
+class ResponseErrorTests: XCTestCase {
+    func testStatusCodeErrorsHaveALocalisedString() {
+        let codeError = ResponseError.statusCode(404)
```

Comment:
You can test this by casting to `Error` here

Reply:
Yep